2022/24xxx/CVE-2022-24727.json

```diff
@@ -2,7 +2,7 @@
     "CVE_data_meta": {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'weblate', 'version': {'version_data': [{'version_value': '< 4.11.1'}]}}]}, 'vendor_name': 'WeblateOrg'}]}}